tests/main.cpp

```diff
@@ -0,0 +1,18 @@
+#include "./lp_hash_table_test.h"
+#include <assert.h>
+#include <iostream>
+
+int main() {
+  std::cout << "Hash Table" << std::endl;
+  bool res = test_hash_table();
+  assert(res == true);
+
+  res = test_many_key_insertion();
+  assert(res == true);
+
+  std::cout << "\x1b[0;32m Hash table tests passed! \x1b[0m" << std::endl;
```

Comment:
IIRC this colors the message. Funny but please don't do that because it's part of another issue and we don't want to spoil it. :P

Reply:
My bad sorry

Reply:
No problem.